.github/workflows/stale.yml

```diff
@@ -51,4 +51,3 @@ jobs:
             testing instructions, is it up to date with master, and it is still valid.
             Feel free to close this PR if you think it's not valid anymore — if you
             do, please add a brief explanation.</p>
-          operations-per-run: 2
```

Comment:
@brbrr You had added this parameter when we first introduced the Stale bot to the repo in #9835. Do you remember if there was something that broke when we didn't have that limit?